app/src/main/res/layout/task_list.xml

```diff
@@ -12,14 +15,16 @@
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintLeft_toLeftOf="parent">
         <RadioButton
+            android:id="@+id/linearLayoutToDoRadioButton"
```

Comment:
This is a very large ID. Just radioButton_todo would've been fine and according to the conventions.

FYI, two different layout files can have the same IDs. Won't cause any collisions.

Reply:
Allright